xml/hwbp_registers.xml

```diff
@@ -220,6 +220,19 @@
         hold all valid virtual addresses but it need not be capable of holding
         other values.
 
+        If the A extension is supported, then trigger behavior is as follows
+        for the load and store bits:
+        \begin{tightlist}
+        \item LR instructions are loads
+        \item successful SC instructions are stores
+        \item it is \unspecified\ whether failing SC instructions are stores or not
+        \item Each AMO instruction is both a load and a store
+        \end{tightlist}
+
+        If the destination register of any load or AMO is x0 then it is
```

Comment:
x0 -> \Rzero

Reply:
I was trying to be consistent with the A extension:
> More complex implementations might also implement AMOs at memory controllers, and can optimize away
fetching the original value when the destination is x0.

In some situations (not this particular case), that prevents confusion such as Zicsr's "For both CSRRS and CSRRC, if rs1=x0, then the instruction will not write to the CSR at all" (rs1=zero would introduce ambiguity between the register number and the register value).  Rather than sometimes calling it x0 (to prevent confusion) and sometimes calling it zero (when there is no confusion), it's best to consistently use the unambiguous name.  All the architecture documents I've seen call it x0 consistently.

Reply:
If x0 is the standard, then I'm happy with a change to the macro to render it as x0. Optionally we could change the name of the macro too.